activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -314,6 +314,12 @@ def supports_pgcrypto_uuid?
         postgresql_version >= 90400
       end
 
+      # PostgreSQL 9.4 introduces ALTER TABLE ... ALTER CONSTRAINT but it has a bug and fixed in 9.4.2
+      # https://www.postgresql.org/docs/9.4/static/release-9-4-2.html
```

Comment:
Could you move this comment to inside the method? Otherwise it will show as documentation of it.